rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/30_synonyms_delete.yml

```diff
@@ -0,0 +1,57 @@
+setup:
+  - skip:
+      version: " - 8.8.99"
+      reason: Introduced in 8.9.0
+  - do:
+      synonyms.put:
+        synonyms_set: test-get-synonyms
+        body:
+          synonyms_set:
+            - synonyms: "hello, hi"
+              id: "test-id-1"
+            - synonyms: "bye => goodbye"
+              id: "test-id-2"
+
+---
+"Delete synonyms set":
+  - do:
+      synonyms.delete:
+        synonyms_set: test-get-synonyms
+
+  - match:
+      acknowledged: true
+
+---
+"Delete synonyms set - not found":
+  - do:
+      catch: missing
+      synonyms.delete:
+        synonyms_set: unknown-synonym-set
+
+---
+"Delete synonyms set - does not impact other synonym sets":
+  - do:
+      synonyms.put:
+        synonyms_set: test-other-synonyms
+        body:
+          synonyms_set:
+            - synonyms: "hola, hi"
+              id: "test-other-1"
+            - synonyms: "test => check"
+              id: "test-other-2"
+  - do:
+      synonyms.delete:
+        synonyms_set: test-get-synonyms
```

Comment:
Is it possible that `test-get-synonyms` is already deleted by this point, since we delete it on line 18? Should we do another PUT to be on the safe side before deleting here, or use a different synonym set ID in this test?

Reply:
I'd say no, as this is created in setup. I'm expecting setup to be executed before each test case.